keps/sig-cloud-provider/2436-controller-manager-leader-migration/README.md

```diff
@@ -48,7 +48,7 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 - [X] (R) Graduation criteria is in place
 - [X] (R) Production readiness review completed
 - [X] (R) Production readiness review approved
-- [ ] "Implementation History" section is up-to-date for milestone
+- [X] "Implementation History" section is up-to-date for milestone
```

Comment:
Can't comment on the corresponding line, so commenting here.

L165: we just removed support for endpoints/configmaps-based leases
  Can we somehow ensure that we really will be using leases as a lock?
  Do we know if someone was using something else?
  Can we safely remove non-leases stuff?

Reply:
@wojtek-t the primary objective of this KEP is a controlled migration of controllers from binary/A to binary/B.  Given that objective, I don't think it's possible to tighten the acceptable values smaller than the values allowed by the binaries you're migrating.

As the library they use gets tighter, I think this gets fixed by default.  If the v1 configuration API here after 1.26, they won't be able to set a working value other than leases because the library won't function, correct?

Reply:
(a) default is already lease.
(b) I agree we should remove anything other than "lease" from the comments. Will update the KEP and later in the API.

Reply:
> this is more of a "temporary" feature

I don't think this is correct. The feature is the ability to migrate controllers from one kinf of controller manager to another. I agree that the impetus for implementing this is a one time move. However I think other instances of needing this feature will crop up. I am already aware of several other instances.

Reply:
@cheftako - sorry, it was a bit of mental shortcut.
What I meant is not that "we will get rid of this feature in few releases", but rather "a single incarnation of usage of this feature is temporary". In other words, In that sense, the fact that I used "endpoints" last time to migrate something off doesn't mean that I will need to use "endpoints" for the other thing in 5 releases from now.

And us such, I want to start advertising "endpoints" completely and soon force people to actually use leases.

Reply:
We can remove this field in v1 API but keep it in v1beta1

Reply:
> "a single incarnation of usage of this feature is temporary"
+1